### DISCUSSION TITLE
installation de la CLI de symfony

### DISCUSSION THREAD

**Discussion by jordantchougong** [symfony/symfony#43232] [Q&A] [answered]
lorsque je veux installer la CLI de symfony dans ma machine je reussi pas et je suis confronté à un problème.
le problème est le suivant: failed to send request to the HTTP server
![Capture_CLI](https://user-images.githubusercontent.com/71893842/135086743-125967b7-18cd-4b2b-9b7f-58cb51380cbc.PNG)

**Comment by jordantchougong** [comment]
s'il vous plait j'aimerais que vous m'aidiez à résoudre celà.. je sais pas si j'ai mal installé quelque chose

**Reply by michaoj** [reply]
Hi,

The preferred language for those topics is English, so could you please update your message ?

By the way, it seems you cannot download one of the needed files but it's hard to say more without any context : 

- Did you checked the requirements ?
- Did you get the setup from the official symfony site ?
- Do you have a proxy/ an antivirus which can prevernt you from accessing the file over http ? And by the way, do you have internet access on the machine where you wan't to install symfony ?

**Reply by jordantchougong** [reply]
when i want to install the symfony CLI in my machine i have no success and i am faced with a problem.
the problem is: failed to send request to the HTTP server and you can see the attach file and yes i have internet access on my machine and all my proxy or antivirus are off.. i get the setup on the official site

**Comment by stof** [comment, marked as answer]
The Symfony CLI is not open-source but maintained by the Symfony SAS team. So reporting it at https://github.com/symfony/cli/issues seems a better idea.

**Reply by jordantchougong** [reply]
okay
